README.md

```diff
@@ -161,6 +161,9 @@ require('nvim-tree-preview').setup {
   },
   on_open = nil, -- fun(win: number, buf: number) called when the preview window is opened
   on_close = nil, -- fun() called when the preview window is closed
+  watch = {
+    enable_on_hold = false -- if true, update preview on CursorHold instead of immediately on CursorMoved
```

Comment:
Maybe we could change this to `watch.event` which accepts either `"CursorHold"` or `"CursorMoved"` (default to `"CursorMoved"`, and validate in the setup function). Then, we can pass the string directly to `nvim_create_autocmd`. What do you think?